Sources/XiEditor/AppDelegate.swift

```diff
@@ -227,16 +239,12 @@ class AppDelegate: NSObject, NSApplicationDelegate {
                 return
             }
         }
-        let alert = NSAlert()
-        alert.addButton(withTitle: "OK")
-        alert.alertStyle = .informational
-        alert.messageText = message
-        alert.informativeText = info
-        alert.runModal()
+        informationalAlert(title: message, message: info)
+        renameCliToggle()
     }
     
-    @IBAction func removeCli(_ sender: Any) {
-        let cliPath = URL(string: "file:///usr/local/bin/xi")!
+    func removeCli() {
+        let cliPath = URL(fileURLWithPath: "/usr/local/bin/xi")
         var message = "CLI Removed!"
         var info = "The file /usr/local/bin/xi has been deleted."
```

Comment:
I think using a path from the `cliPath` would be a nice little touch here 😄

Reply:
Amended!